### ISSUE TITLE
Compilation error with disposables

### ISSUE THREAD

**Issue by jesusgp22** [react-dnd/react-dnd#664]
Trying to use babel 6 to compile this package results in:

```
ERROR in ./~/disposables/modules/index.js
Module build failed: ReferenceError: [BABEL] C:\...\node_modules\disposables\modules\index.js: Using removed Babel 5 option: C:\...\node_modules\disposables\.babelrc.stage - Check out the corresponding stage-x presets http://babeljs.io/docs/plugins/#presets
```
The disposables repo states that the package is not maintained, is it posible to update this dependency?

**Comment by darthtrevino** [comment]
Are you using yarn to install deps? It's using babel 6 now, and it's functioning correctly in CI.

**Comment by jesusgp22** [comment]
I'm using npm, only method that worked was manually removing the .babelrc file from disposables

**Comment by alexicum** [comment]
Same problem: [Using removed Babel 5 option: base.stage](https://github.com/halt-hammerzeit/webpack-react-redux-isomorphic-render-example/issues/21#issuecomment-284310365)

Is it possible to update disposables to [rx.disposables](https://github.com/Reactive-Extensions/rx.disposables)?

**Comment by lalomartins** [comment]
Having the same problem here. Please note disposables is unmaintained, and their readme recommends switching to rx.disposables.

**Comment by karlis-repsons** [comment]
@darthtrevino - how can you just close this issue before the person even answered your question about yarn? In fact, react-dnd docs show use of npm and me and a list of other people have trouble because of the old disposables.

**Comment by darthtrevino** [comment]
Other people have answered the question, and there was a PR that proposed to fix this issue where the author confirmed they were using npm. I believe the top-level docs were updated to mention yarn, but if you find any other references, feel free to cut a PR or call them out in an issue.

**Comment by gaearon** [comment]
If you have this issue it means you're compiling `node_modules` with Babel.

This is not safe to do for all packages, and more things can break over time if you expect their config formats to match your Babel version.

The easiest fix is to ignore `node_modules` with Babel. For example by speciyfing `exclude: /node_modules/` in `babel-loader` options.

Anyway, I published a "fix" in `disposables@1.0.2`. But you'll keep having this problem with other packages and future Babel versions if you don't fix your build setup.
